opa-audit-operator/files/pod-spec.yaml.jinja2

```diff
@@ -0,0 +1,190 @@
+          verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+        - apiGroups:
+            - config.gatekeeper.sh
+          resources:
+            - configs
+          verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+        - apiGroups:
+            - config.gatekeeper.sh
+          resources:
+            - configs/status
+          verbs:
+            - get
+            - patch
+            - update
+        - apiGroups:
+            - constraints.gatekeeper.sh
+          resources:
+            - '*'
+          verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+        - apiGroups:
+            - policy
+          resourceNames:
+            - gatekeeper-admin
+          resources:
+            - podsecuritypolicies
+          verbs:
+            - use
+        - apiGroups:
+            - status.gatekeeper.sh
+          resources:
+            - '*'
+          verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+        - apiGroups:
+            - templates.gatekeeper.sh
+          resources:
+            - constrainttemplates
+          verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+        - apiGroups:
+            - templates.gatekeeper.sh
+          resources:
+            - constrainttemplates/finalizers
+          verbs:
+            - delete
+            - get
+            - patch
+            - update
+        - apiGroups:
+            - templates.gatekeeper.sh
+          resources:
+            - constrainttemplates/status
+          verbs:
+            - get
+            - patch
+            - update
+        - apiGroups:
+            - admissionregistration.k8s.io
+          resourceNames:
+            - gatekeeper-validating-webhook-configuration
+          resources:
+            - validatingwebhookconfigurations
+          verbs:
+            - create
+            - delete
+            - get
+            - list
+            - patch
+            - update
+            - watch
+containers:
+  - envConfig:
+      POD_NAMESPACE:
+        field:
+          path: metadata.namespace
+          api-version: v1
+      POD_NAME:
+        field:
+          path: metadata.name
+          api-version: v1
+    imageDetails:
+      imagePath: {{ image_path }}
+    ports:
+      - containerPort: 8888
+        name: metrics
+        protocol: TCP
+      - containerPort: 9090
+        name: healthz
+        protocol: TCP
+    imagePullPolicy: {{ imagePullPolicy }}
+    name: {{ app_name }}-audit
```

Comment:
I wouldn't bother including app_name in the container name. The container is already going to be scoped to a pod that includes the app name anyway.

Reply:
:+1: